mime_test.go

```diff
@@ -568,7 +587,7 @@ func TestMailYakBuildMime_withAttachments(t *testing.T) {
 			// Itterate over the mime parts, look for attachments
 			for {
 				p, err := mr.NextPart()
-				if err == io.EOF {
+				if errors.Is(err, io.EOF) {
```

Comment:
Thank you for review, your kindness is much appreciated!

This OK, I expected something like this with such bad shortcut on my side. Unfortunately handling message with no attachment became much harder and I resorted to `net/mail` package to do full header/body parsing. I hope this will work and acceptable.